tests/sentry/snuba/metrics/test_metrics_layer/test_metrics_enhanced_performance.py

```diff
@@ -385,6 +385,13 @@ def test_query_with_has_condition(self):
                 value=value,
             )
 
+        # We also store a metric without the transaction tag.
```

Comment:
What is this for?

Reply:
I wanted to make a more meaningful test to add an additional transaction without the `transaction` tag.